tools/codegen/pkg/swaggerdocs/generator.go

```diff
@@ -12,6 +12,16 @@ import (
 
 // Options contains the configuration required for the swaggerdocs generator.
 type Options struct {
+	// Enabled indicates whether the swaggerdocs generator is enabled or not.
```

Comment:
i notice that we are using `Enabled` here in the comments which matches the value in the struct, but in the other locations (in this file and in `tools/codegen/pkg/generation/types.go`) we are using the serializable version of the name. i think for clarity these should be similar format, i'm not sure which the correct version though.

i notice in `tools/codegen/pkg/generation/generator.go` we use the name with the same capitalization in the comments as in the struct. perhaps these should all be capitalized the same?

Reply:
I've moved everything to the normal godocs style which matches the field names. I was thinking the config file should have the serializable names as the docs then match what the users would see, but we aren't going to generate docs from this so it's ok, they should be consistent now

Reply:
makes sense, i was just curious because of the difference